public/css/custom-style.css

```diff
@@ -759,4 +759,8 @@ ul {
 .slider-transform {
   transform: translate3d(-200%, 0px, 0px); 
   transition-duration: 350ms;
+}
+
+.margin-le{
```

Comment:
let's use bootstrap margin/padding classes instead, check this docs https://getbootstrap.com/docs/5.3/utilities/spacing/#margin-and-padding, in this case it would be .ml-4